.github/workflows/python_build.yaml

```diff
@@ -0,0 +1,214 @@
+name: Offline Installation Archive
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
Remove this now that it works. We may want to trigger this on release (completion of python_release workflow)

Reply:
+1 for triggering this on release.

Reply:
If we trigger on release should we make this a `workflow_call` rather than a workflow dispatch? That way we use https://github.com/sift-stack/sift/blob/main/.github/workflows/python_ci.yaml upstream and do all the last minute sanity checks before committing to building